project/Snapper.sln

```diff
@@ -13,6 +13,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Snapper.Tests", "Tests\Snap
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Snapper.Nunit.Tests", "Tests\Snapper.Nunit.Tests\Snapper.Nunit.Tests.csproj", "{1E140E12-9738-402A-984C-B59AD0382E6E}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Snapper.MSTest.Tests", "Tests\Snapper.MSTest.Tests\Snapper.MSTest.Tests.csproj", "{291D131A-0157-41C6-B72D-CF21CF9056F8}"

```

Comment:
Rather than making a new test project for every new test framework I've just made a new generic test project. In this PR https://github.com/theramis/Snapper/pull/32

Could you please add your tests there?